cts/lab/CTStests.py

```diff
@@ -848,189 +848,6 @@ def errors_to_ignore(self):
-        lpc = 0
-        failed = 0
-        delay = 2
-        done = time.time() + self._env["loop-minutes"]*60
-        self.find_hae_resources(node)
-
-        clone_max = len(self._env["nodes"])
-        while time.time() <= done and not failed:
-            lpc = lpc + 1
-
-            self.change_state(node, self.r_dlm, "Stopped")
-            if not self.wait_on_state(node, self.r_dlm, 0):
-                self.failure("%s did not go down correctly" % self.r_dlm)
-                failed = lpc
-
-            self.change_state(node, self.r_dlm, "Started")
-            if not self.wait_on_state(node, self.r_dlm, clone_max):
-                self.failure("%s did not come up correctly" % self.r_dlm)
-                failed = lpc
-
-            if not self.wait_on_state(node, self._r_o2cb, clone_max):
-                self.failure("%s did not come up correctly" % self._r_o2cb)
-                failed = lpc
-
-            for fs in self._r_ocfs2:
-                if not self.wait_on_state(node, fs, clone_max):
-                    self.failure("%s did not come up correctly" % fs)
-                    failed = lpc
-
-        if failed:
-            return self.failure("iteration %d failed" % failed)
-        return self.success()
-
-AllTestClasses.append(HAERoleTest)
-
-
-class HAEStandbyTest(HAETest):
-    '''Set up a custom test to cause quorum failure issues for Andrew'''
-    def __init__(self, cm):
-        HAETest.__init__(self,cm)
-        self.name = "HAEStandbyTest"
-
-    def change_state(self, node, resource, target):
-        (rc, _) = self._rsh(node, "crm_standby -V -l reboot -v %s" % (target))
-        return rc
-
-    def __call__(self, node):
-        self.incr("calls")
-
-        lpc = 0
-        failed = 0
-        done = time.time() + self._env["loop-minutes"]*60
-        self.find_hae_resources(node)
-
-        clone_max = len(self._env["nodes"])
-        while time.time() <= done and not failed:
-            lpc = lpc + 1
-
-            self.change_state(node, self.r_dlm, "true")
-            if not self.wait_on_state(node, self.r_dlm, clone_max-1):
-                self.failure("%s did not go down correctly" % self.r_dlm)
-                failed = lpc
-
-            self.change_state(node, self.r_dlm, "false")
-            if not self.wait_on_state(node, self.r_dlm, clone_max):
-                self.failure("%s did not come up correctly" % self.r_dlm)
-                failed = lpc
-
-            if not self.wait_on_state(node, self._r_o2cb, clone_max):
-                self.failure("%s did not come up correctly" % self._r_o2cb)
-                failed = lpc
-
-            for fs in self._r_ocfs2:
-                if not self.wait_on_state(node, fs, clone_max):
-                    self.failure("%s did not come up correctly" % fs)
-                    failed = lpc
-
-        if failed:
-            return self.failure("iteration %d failed" % failed)
-        return self.success()
-
-AllTestClasses.append(HAEStandbyTest)
-
-
```

Comment:
You can also drop the hae references in CIB.py

Reply:
Removed.